pages/static/data-updates.html

```diff
@@ -5,6 +5,30 @@
 
 <h1>Data Updates</h1>
 
+<h2 id="2024-04-23">April 23rd, 2024</h2>
+<p>
+    New data from BU, HTAPP, CHOP, OHSU, and Standford is available as part of
+    Release 5.1:
+</p>
+
+<ul>
+    <li>
+        Level 1-2, level 3, and auxiliary 10X Visium data for 18 new
```

Comment:
Thank you, @onursumer ! Very minor, but can we modify the release notes just a tad? 

Level 1-3 and auxiliary 10X Visium data for 18 new participants
Level 2-3 CODEX data for 20 new participants
Level 1-2 Electron Microscopy data for 12 new participants
Level 2 H&E data for 50 new participants
Level 1 LC-MS/MS data for 7 new participants
Level 1 LC-MS3 data for 7 new participants
Level 3-4 MIBI data for 32 new participants
Level 1, 3, and 4 Mass Spectrometry data for 7 new participants
Level 1 Shotgun MS (lipidomics) data for 7 new participants
Level 4 Imaging data for 7 new participants